src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -0,0 +1,2085 @@
+5. **ACCESSIBLE**: technically this is the same scope as the user's tracker search organisation units.
+6. **ALL**: the name ALL makes perfect sense if we are dealing with a super user. For super users, this scope means the entire organisation unit available in the system. However, for non-super users, ALL boils down to ACCESSIBLE organisation units.
+
+It makes little sense to pass these modes at the time of tracker import operations. Because when writing tracker data, each of the objects need to have a specific organisation unit attached to them. The system will then ensure if each of the mentioned organisation units fall under the CAPTURE scope. If not the system will simply reject the write operation.
+
+
+  * **Explain how they relate to ownership - Link to Program Ownership**
+
+### Program Ownership
+
+<!--DHIS2-SECTION-ID:webapi_nti_ownership-->
+
+  * Describe the idea about Program Ownership
+  * Temporary ownership
+  * Program Ownership and Access Level - Link to Access Level
+
+### Access Level
+
+<!--DHIS2-SECTION-ID:webapi_nti_access_level-->
+
+DHIS2 treats Tracker data with extra level of protection. In addition to the standard feature of metadata and data protection through sharing settings, Tracker data are shielded with various access level protection mechanisms. Currently there are 4 access levels: Open, Audited, Protected and Closed.
+
+These access levels are triggered only when users try to interact with data outside their capture scope. If request to data is within the capture scope, DHIS2 applies standard metadata and data sharing protection. To access data outside capture scope, but within search scope, users need to pass through extra level of protection or what we call in DHIS2 concept of “breaking the glass”. The concept is, accessing outside data needs to be justified, has consequence and will be audited for others to see it.  The way to configure the level of consequence of breaking the glass is by setting Program’s access level to either Open, Audited, Protected or Closed.
+
+1. Open: as the name implies, accessing data outside capture scope is possible without any justification or consequence. It is as if the data is within the capture scope. However, it is not possible to modify data captured by another org unit irrespective of the access level.
+2.	Audited: this is the same as Open access level. The difference here is that the system will automatically put audit log entry on the data being accessed by the specific user.
+3.	Protected: this takes audited access protection one level up. This time users need to provide a justification why they are accessing the data at hand. The system will then put a log of both the justification and access audit.
```

Comment:
should we explain how to pass that justification? Is there a property in the request for that?

Reply:
Will leave that for the program ownership section that Ameen is working on. Will leave this comment here so we can link them

Reply:
https://github.com/dhis2/dhis2-docs/pull/682#pullrequestreview-641091307

This should cover it :)